src/components/Calendar.tsx

```diff
@@ -19,6 +19,7 @@ import FullCalendarBox from "./FullCalendarBox";
 import EventDetail from "./EventDetail";
 import initialState from "../calendar/initialCalendarState";
 import ProjectDashboard from "./ProjectDashboard";
+import ReservationPage from "./ReservationPage";
```

Comment:
just continuing the semantic idea mentioned on the `types.ts` file
```suggestion
import ReservationForm from "./ReservationForm";
```

Reply:
all uses of ReservationPage changed to ReservationForm